include/functions.h

```diff
@@ -109,10 +109,10 @@ void FaultDrawer_SetFontColor(u16 color);
 void FaultDrawer_SetCharPad(s8 padW, s8 padH);
 void FaultDrawer_SetCursor(s32 x, s32 y);
 void FaultDrawer_FillScreen(void);
-FaultDrawer* FaultDrawer_FormatStringFunc(FaultDrawer* arg, const char* str, s32 count);
+FaultDrawer* FaultDrawer_FormatStringFunc(FaultDrawer* arg, char* str, s32 count);
 void FaultDrawer_VPrintf(char* str, char* args);
-void FaultDrawer_Printf(const char* fmt, ...);
-void FaultDrawer_DrawText(s32 x, s32 y, const char* fmt, ...);
+void FaultDrawer_Printf(char* fmt, ...);
+void FaultDrawer_DrawText(s32 x, s32 y, char* fmt, ...);
 void FaultDrawer_SetDrawerFB(void* fb, u16 w, u16 h);
```

Comment:
This is the one section I'm not super happy about, I'll keep trying to re-work it.